monkey/infection_monkey/utils/windows/hidden_files.py

```diff
@@ -0,0 +1,71 @@
+HIDDEN_FILE = "%temp%\\monkey-hidden-file"
+HIDDEN_FILE_WINAPI = "%temp%\\monkey-hidden-file-winAPI"
+HIDDEN_FOLDER = "%temp%\\monkey-hidden-folder"
```

Comment:
Not sure we'll get the impact we want from touching %temp%. We should impact user folder.